webapp/packages/plugin-data-viewer/src/ContainerDataSource.ts

```diff
@@ -217,6 +182,58 @@ export class ContainerDataSource extends ResultSetDataSource<IDataContainerOptio
     return prevResults;
   }
 
+  getResultId(prevResults: IDatabaseResultSet[], context: IConnectionExecutionContextInfo) {
```

Comment:
seems like that this new methods used only once, so what is purpose for them to exists? They all public what is the reason for it?